app/controllers/admin/court_applications_controller.rb

```diff
@@ -1,8 +1,32 @@
 module Admin
   class CourtApplicationsController < Admin::ApplicationController
+    def index
+      @court_applications = if params[:query].present?
+                              CourtApplication.search(params[:query]).order(created_at: :desc).page(params[:page])
+                            else
+                              CourtApplication.order(created_at: :desc).page(params[:page])
+                            end
```

Comment:
if you want to avoid duplication you can try:

```
scope = CourtApplication.order(created_at: :desc)
scope = scope.search(params[:query]) if params[:query].present?
@court_applications = scope.page(params[:page])
```